app/scripts/services/util.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module('confRegistrationWebApp')
+    .factory('util', function util() {
+        var service = {
+            isUndefinedOrNull: function (obj) {
```

Comment:
@OzzieOrca  We can't use _.isEmpty as it does not work with numbers. For example, _isEmpty(55) returns true.